### PR TITLE
Feat:Completed patient form

### PR DESCRIPTION
**Pull request by rafael-ramirez-orte**
# Description

Implementation of an alert, form refresh and connection to the backend was done. 

### Screenshots, Videos, Gifs.

image

# Checklist:

- [ ] Pull request owner assigned
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have pulled / merged main/develop into this branch

[IL-52]

### CODE REVIEW

File: navigation/MainStack.jsx [hunk 1]
```diff
@@ -19,6 +18,8 @@ const MainStack = () => {
                     name= 'PatientRegister'
                     component={PatientRegisterPage}
                 />
+
+
```
Comment (JavierRuvalcaba) [inline navigation/MainStack.jsx]: remove the blank spaces

File: navigation/MainStack.jsx [hunk 2]
```diff
@@ -19,6 +18,8 @@ const MainStack = () => {
                     name= 'PatientRegister'
                     component={PatientRegisterPage}
                 />
+
+            
             </Stack.Navigator>
         </NavigationContainer>
 )
```
Comment (JavierRuvalcaba) [inline navigation/MainStack.jsx]: fix the indentation

File: page/LoginPage.jsx [hunk 3]
```diff
@@ -11,6 +11,7 @@ const LoginPage = ({navigation}) => {
                 navigation.navigate('PatientRegister')
             }}
             />
+
```
Comment (JavierRuvalcaba) [inline page/LoginPage.jsx]: remove the blank space and fix the indentation for the Button component

File: page/PatientRegisterPage.jsx [hunk 4]
```diff
@@ -1,18 +1,177 @@
...
+  const cleanForm = () => {
+    setValue("name", "");
+    setValue("typeOfCancer", "");
+    setValue("services", "");
+    setValue("palliative", "");
+    setValue("age", "");
+  };
+  const { patientRequest } = useFetch();
+  const onSubmit = async (data) => {
+    console.log(data);
```
Comment (JavierRuvalcaba) [inline page/PatientRegisterPage.jsx]: remove this console.log

File: page/PatientRegisterPage.jsx [hunk 5]
```diff
@@ -1,18 +1,177 @@
...
+  } = useForm();
+  const cleanForm = () => {
+    setValue("name", "");
+    setValue("typeOfCancer", "");
+    setValue("services", "");
+    setValue("palliative", "");
+    setValue("age", "");
+  };
+  const { patientRequest } = useFetch();
+  const onSubmit = async (data) => {
```
Comment (JavierRuvalcaba) [inline page/PatientRegisterPage.jsx]:
```suggestion
  const {
    control,
    handleSubmit,
    setValue,
    formState: { errors },
  } = useForm();
  
  const cleanForm = () => {
    setValue("name", "");
    setValue("typeOfCancer", "");
    setValue("services", "");
    setValue("palliative", "");
    setValue("age", "");
  };
  
  const { patientRequest } = useFetch();
  
  const onSubmit = async (data) => {
```

**Review by JavierRuvalcaba — CHANGES_REQUESTED** [review]

**Review by JavierRuvalcaba — APPROVED** [review]

**Review by isakk7 — APPROVED** [review]

**Pull request merged**